index.rst

```diff
@@ -100,73 +110,268 @@ Data Sets Types and Goals
+
+2. Missing Features
+
+   Three important data features are missed in all of the datasets described above, as they are generically missing all datasets that are subsets of HSC SSP PDR1/2 and RC2:
+
+      - Differential chromatic refraction (HSC has an atmospheric dispersion corrector).
+
+      - LSST-like wavefront sensors (HSC's are too close to focus to be useful for learning much about the state of the optical system).
+
+      - Crowded stellar fields.
+
+   A (not yet identified) DECam dataset could potentially address all of these issues, but characterizing the properties of DECam at the level already done for HSC may be difficult, and would probably be necessary to fully test the DM algorithms for which DCR and wavefront sensors are relevant (e.g., physically-motivated PSF modeling).  Many non-PDR1/2+RC2 HSC datasets do include more interesting variability or crowded fields, so it *might* be most efficient to just add one of these to our test data suite, and defer some testing of DCR or wavefront-sensor algorithms until data from ComCam or even the full LSST camera are available.
+
+DRP Summary
+===========
+
+CI, SMALL, MEDIUM, and LARGE datasets exist suitable for significant amount of Science Pipelines performance monitoring.  The addition of a dataset on a crowded field would help exercise a key portion of the Science Pipelines that currently is uncertain.  Technical investigations of (1) using wavefront-sensor data and (2) a system without an ADC may wait until commissioning data is available from ComCam or the full LSSTCam.
+
+=================
+AP Test Datasets
+=================
+Summary recommendations:
+  1. use a subset of HiTS for quick turnaround processing, smoke tests, etc.  DONE.
+  2. use the DECam Bulge survey for crowded field tests.  IN PROGRESS.
+  3. Select a subset of HSC SSP PDR1 vs PDR2.  TICKET OPEN.
+  4. use a DES Deep SN field for large-scale processing.
+
+Desiderata for AP testing:
+  - tens of epochs per filter per tract in order to construct templates for image differencing and to characterize variability
+  - the ability to exercise as many aspects of LSST pipelines and data products as possible
+  - public availability (so that we can feely recruit various LSST stakeholders)
+  - potential for enabling journal publications (both technical and scientific) so that various stakeholders beyond LSST DM may have direct interest in contributing tools and analysis.
+  - We should have datasets from at least two different cameras, so that we can isolate effects of LSST pipeline performance from camera-specific details (e.g., ISR, PSF variations) that impact the false-positive rate
```

Comment:
Remove "We should have", so just "- datasets from at least two different cameras..."